browser-test/src/trusted_intermediary.test.ts

```diff
@@ -573,7 +585,7 @@ test.describe('Trusted intermediaries', () => {
     await tiDashboard.clickOnViewApplications()
     await page.click('#ti-clients-link')
 
-    expect(await page.innerText('#add-client')).toContain('Add Client')
+    expect(await page.innerText('#ti-search-form')).toContain('Search by name')
```

Comment:
suggestion (non-blocking):  rather than verifying the text, you could verify that the innerHtml has the id `name-search`.  This is so that we don't have to update this test if the text ever changes.